### PR TITLE
Kotlin 1.0.0 beta 1038

### PR DESCRIPTION
**Pull request by fboldog**

### CODE REVIEW

File: Library/Formula/kotlin-compiler.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class KotlinCompiler < Formula
   desc "Statically typed programming language for the JVM"
   homepage "https://kotlinlang.org/"
-  url "https://github.com/JetBrains/kotlin/releases/download/build-0.14.449/kotlin-compiler-0.14.449.zip"
-  sha256 "bbb55c84669b6d920dc89beab811b26dfbdbeb95d71e1d6d3384fbfbcd48ec70"
+  url "https://github.com/JetBrains/kotlin/releases/download/build-1.0.0-beta-1038/kotlin-compiler-1.0.0-beta-1038.zip"
```
Comment (DomT4) [inline Library/Formula/kotlin-compiler.rb]: Unless there's a solid reason, developmental versions shouldn't replace stable versions here. You can however add a `devel` block for this purpose.

Comment (fboldog) [inline Library/Formula/kotlin-compiler.rb]: Ok, thanks.

File: Library/Formula/kotlin-compiler.rb [hunk 2]
```diff
@@ -1,8 +1,15 @@
 class KotlinCompiler < Formula
   desc "Statically typed programming language for the JVM"
   homepage "https://kotlinlang.org/"
-  url "https://github.com/JetBrains/kotlin/releases/download/build-0.14.449/kotlin-compiler-0.14.449.zip"
-  sha256 "bbb55c84669b6d920dc89beab811b26dfbdbeb95d71e1d6d3384fbfbcd48ec70"
+  stable do
+    url "https://github.com/JetBrains/kotlin/releases/download/build-0.14.449/kotlin-compiler-0.14.449.zip"
+    sha256 "bbb55c84669b6d920dc89beab811b26dfbdbeb95d71e1d6d3384fbfbcd48ec70"
+  end
```
Comment (bfontaine) [inline Library/Formula/kotlin-compiler.rb]: You don’t need a `stable` block here; the top-level `url` is always stable.

**Comment by DomT4** [comment]
Closed via https://github.com/Homebrew/homebrew/commit/8119332939bc2edf858b9acccd24414deb14fe52.

**Pull request closed without merging**